### PR TITLE
first version of all the stages using the builder

### PR DESCRIPTION
**Pull request by melisabok**
depends on https://github.com/iesl/openreview-py/pull/216

### CODE REVIEW

File: venues/ICLR.cc/2019/Workshop/DeepGenStruct/python/config.py [hunk 1]
```diff
@@ -18,8 +18,11 @@ def get_conference(client):
     'website': 'https://deep-gen-struct.github.io/index.html',
     'location': 'New Orleans, Louisiana, United States'
     })
+    builder.set_authorpage_header({
+        'title': 'Author Console',
+        'instructions': 'This are the instructions',
+        'schedule': 'To be Defined'
+    })
```
Comment (pmandler) [inline venues/ICLR.cc/2019/Workshop/DeepGenStruct/python/config.py]: Instructions should be blank?  
schedule should capitalize 'be' or lower 'defined'

Comment (melisabok) [reply venues/ICLR.cc/2019/Workshop/DeepGenStruct/python/config.py]: Fixed

File: venues/ICLR.cc/2019/Workshop/DeepGenStruct/python/review-assignment-stage.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+print ("Set global committee members")
+conference.set_authors()
+conference.set_program_chairs(['pc@mail.com'])
+conference.set_area_chairs(['mbok@cs.umass.edu'])
+conference.set_reviewers(['pmandler@cs.umass.edu'])
+
+print ("Create some assignments")
+conference.set_assignment('mbok@cs.umass.edu', 1, is_area_chair = True)
+conference.set_assignment('mandler@cs.umass.edu', 1, is_area_chair = False)
```
Comment (pmandler) [inline venues/ICLR.cc/2019/Workshop/DeepGenStruct/python/review-assignment-stage.py]: Is this just checked in as a placeholder?  It seems like code that will either be done in Jupyter notebook or read in assignments from a spreadsheet.  I don't think there are area chairs for this workshop.

Comment (melisabok) [reply venues/ICLR.cc/2019/Workshop/DeepGenStruct/python/review-assignment-stage.py]: It is an example, I can remove this code.

Comment (melisabok) [reply venues/ICLR.cc/2019/Workshop/DeepGenStruct/python/review-assignment-stage.py]: I removed the area chair

**Comment by melisabok** [comment]
@pmandler can we merge this?

**Pull request merged**